spec/requests/surveys_controllers_spec.rb

```diff
@@ -9,4 +9,13 @@
       expect(response).to have_http_status(:ok)
     end
   end
+
+  describe 'qrcode' do
+    let(:survey) { create :survey }
+
+    it 'returns 200 status code' do
+      get qrcode_survey_path(survey)
+      expect(response).to have_http_status(:ok)
```

Comment:
Maybe also test that it has a qr code and the short url

Reply:
how?

Reply:
Best would be view spec. We already have one, you can copy from there